pkg/handler/eval_cache.go

```diff
@@ -59,6 +62,37 @@ func (ec *EvalCache) Start() {
 	}()
 }
 
+func (ec *EvalCache) GetByTags(tags []string) []*entity.Flag {
+	ec.mapCacheLock.RLock()
+	defer ec.mapCacheLock.RUnlock()
+
+	results := map[uint]*entity.Flag{}
+	for _, t := range tags {
+		f, ok := ec.tagCache[t]
+		if ok {
+			results = merge(results, f)
+		}
+	}
+
+	return values(results)
+}
+
+func values(m map[uint]*entity.Flag) []*entity.Flag {
```

Comment:
I think the function names `values` and `merge` are too generic inside the handler package, and these two functions are simple enough and I think it's better to just inline into the GetByTags function.